decidim-budgets/app/controllers/decidim/budgets/admin/projects_controller.rb

```diff
@@ -161,6 +162,28 @@ def update_selected
           end
         end
 
+        def update_budget
+          enforce_permission_to :update, :project, project: sample_project
+          ::Decidim::Budgets::Admin::UpdateProjectsBudget.call(reference_budget, project_ids) do
+            on(:invalid_project_ids) do
+              flash.now[:alert] = t(
+                "projects.update_budget.select_a_project",
+                scope: "decidim.budgets.admin"
+              )
```

Comment:
Usually we keep the I18n on the same line ( check the line 33, 38, 56, 61 in the same file. 

```suggestion
              flash.now[:alert] = t("projects.update_budget.select_a_project",scope: "decidim.budgets.admin")
```